sdk/src/protocols/stakewise/v2/index.ts

```diff
@@ -0,0 +1,48 @@
+import { NotFoundError } from "../../../errors"
+import ethPools from "./_ethPools"
+import { EthPool } from "./types"
+import { eth as univ3_eth } from "../../uniswap_v3/index"
+import { EthToken } from "../../uniswap_v3/types"
+import { allow } from "zodiac-roles-sdk/kit"
+import { c } from "zodiac-roles-sdk"
+import { contracts } from "../../../../eth-sdk/config"
+
+const findPool = (pools: readonly EthPool[], nameOrAddress: string) => {
+  const nameOrAddressLower = nameOrAddress.toLowerCase()
+  const pool = pools.find(
+    (pool) =>
+      pool.name.toLowerCase() === nameOrAddressLower ||
+      pool.address.toLowerCase() === nameOrAddressLower
+  )
+  if (!pool) {
+    throw new NotFoundError(`Pool not found: ${nameOrAddress}`)
+  }
+  return pool
+}
+
+export const eth = {
+  deposit: async ({
+    targets
+  }: {
+    targets: (EthPool["name"] | EthPool["address"])[]
+  }) => {
+    const univ3Permissions = await Promise.all(
+      targets.flatMap((target) =>
```

Comment:
```suggestion
      targets.map((target) =>
```

The array is generally flat list of promises, so `flatMap` has no extra effect over simple `map`